pages/linux/treetime.md

```diff
@@ -0,0 +1,20 @@
+# treetime
+
+> TreeTime provides routines for ancestral sequence reconstruction and inference of molecular-clock phylogenies.
+> More information: <https://manned.org/treetime>.
```

Comment:
```suggestion
> More information: <https://treetime.readthedocs.io/en/latest/tutorials.html>.
```
Just a small suggestion, I found their official documentation can you update your link to this?